src/annotator/util/shadow-root.js

```diff
@@ -30,6 +30,7 @@ function loadStyles(shadowRoot) {
  */
 export function createShadowRoot(container) {
   if (!container.attachShadow) {
+    // stopEventPropagation(container);
```

Comment:
@robertknight, what should we do if shadow DOMs are not available? Should we still stop the propagation of events? It all depends on the structure of the page.

Reply:
Can you see any problems caused by stopping propagation of events on the `container` element instead of the shadow root?

Reply:
On a second thought, I don't see any problems.

Propagation of events happens parent -> child and child -> parent.

```
A host element
        |
    - Stop propagation -
        |
        | -> annotator element
```

Reply:
OK, let's do this then.

Reply:
My only concerns was if the `container` element was ever host element.

I can see that `createShadowRoot` is only used 3 times on the following elements created by the annotator:
- `hypothesis-adder`
- `hypothesis-sidebar`
- `hypothesis-notebook`

Hence, I there is no risk of affecting the host page in anyway.

Reply:
OK, I have uncommented this line.